app/frontend/src/features/BugReport.tsx

```diff
@@ -129,53 +151,95 @@ export default function BugReport({
         // won't close when clicking outside
         disableBackdropClick
       >
-        <form onSubmit={onSubmit}>
-          <DialogTitle id="bug-reporter">{REPORT}</DialogTitle>
-          <DialogContent>
-            {error && <Alert severity="error">{error.message}</Alert>}
-            <DialogContentText>{WARNING}</DialogContentText>
-            <TextField
-              id="bug-report-subject"
-              className={classes.field}
-              label={BUG_DESCRIPTION_NAME}
-              helperText={BUG_DESCRIPTION_HELPER}
-              name="subject"
-              inputRef={register({ required: true })}
-              fullWidth
-            />
-            <TextField
-              className={classes.field}
-              id="bug-report-description"
-              label={PROBLEM_NAME}
-              helperText={PROBLEM_HELPER}
-              name="description"
-              inputRef={register({ required: true })}
-              fullWidth
-              multiline
-              rows={4}
-              rowsMax={6}
-            />
-            <TextField
-              className={classes.field}
-              id="bug-report-results"
-              defaultValue=""
-              label={EXPECT_NAME}
-              helperText={EXPECT_HELPER}
-              name="results"
-              inputRef={register}
-              fullWidth
-              multiline
-              rows={4}
-              rowsMax={6}
-            />
-          </DialogContent>
-          <DialogActions>
-            <Button type="submit" loading={isLoading} onClick={onSubmit}>
-              {SUBMIT}
-            </Button>
-            <Button onClick={handleClose}>{CANCEL}</Button>
-          </DialogActions>
-        </form>
+        <DialogTitle id="bug-reporter">{REPORT}</DialogTitle>
+        {type === undefined ? (
```

Comment:
```suggestion
        {type === "initial" ? (
```
If following the above suggestion